backend/app/assets/stylesheets/spree/backend/components/_navigation.scss

```diff
@@ -30,6 +70,17 @@ nav.menu {
   border-right: $border-sidebar;
   background: $color-sidebar-bg;
   z-index: $z-index-navbar-flyout;
+  @include media($desktop) {
+    width: $width-sidebar-collapsed;
+  }
+  @include media($tablet) {
+    width: $width-sidebar;
+    position: fixed;
+    margin: 0 0 0 -200px;
```

Comment:
Also would prefer to use `translateX()` to move it over but not too concerned about the margin.